packages/react-native-sdk/docusaurus/docs/reactnative/03-core/03-call-and-participant-state.mdx

```diff
@@ -69,6 +69,7 @@ Here is an excerpt of the available call state hooks:
 | `useCallCreatedAt`                   | The time the call was created.                                                                                |
 | `useCallCreatedBy`                   | The user that created the call.                                                                               |
 | `useCallCustomData`                  | The custom data attached to the call.                                                                         |
+| `useCallDuration`                    | The call duration since the start of the call in seconds.                                                     |
```

Comment:
It would be good to add this to the React docs too. I know it is annoying to have the same content spread in three places :)